tardis/tests/tests_slow/conftest.py

```diff
@@ -39,6 +42,31 @@ def pytest_unconfigure(config):
     shutil.rmtree(config.option.tempdir)
 
 
+@pytest.hookimpl(tryfirst=True, hookwrapper=True)
+def pytest_runtest_makereport(item, call):
+    # execute all other hooks to obtain the report object
+    outcome = yield
+    report = outcome.get_result()
+
+    dokuwiki_url = item.config.dokureport.dokuwiki_url
+    # TODO: remove hardcoded name w7 after fixture parametrization
+    plots = glob.glob(os.path.join(item.config.option.tempdir, "w7", "*.png"))
```

Comment:
The test method has saved its plots in this directory, so all of them are picked up.